openshift_metrics/tests/test_utils.py

```diff
@@ -13,11 +13,19 @@
 from requests.exceptions import ConnectionError
 import tempfile
 from unittest import TestCase, mock
+from decimal import Decimal
 
 from openshift_metrics import utils, invoice
 import os
 from datetime import datetime, UTC
 
+RATES = invoice.Rates(
+    cpu = Decimal("0.013"),
```

Comment:
Is there a reason the test is manually filling the values? doesn't this mean when rates change you will manually need to update the test?

Reply:
@joachimweyl Couple of things:
1. I need to set it statically because if the rates change then all the known results in my tests would need to be updated. What I am testing is, given some rate, do we get the expected results for various combinations of workloads, conditions etc.
2. Even if I were to get these rates from nerc-rates repo for tests, they will still be static because we get rates for a specific month. If the rates change then those would be effective from the month it changed.

To be clear, these tests aren't specifically testing the mechanism of getting these rates, but rather they just need some rates to proceed with whatever they were already testing.